src/test/java/integration/issue974Test.java

```diff
@@ -0,0 +1,37 @@
+package integration;
+
+import com.codeborne.selenide.SelenideConfig;
+import com.codeborne.selenide.SelenideDriver;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+
+
+class issue974Test extends ITest {
+  private SelenideDriver driver;
```

Comment:
You can initialize it directly there:
`SelenideDriver driver = new SelenideDriver(new SelenideConfig().baseUrl(getBaseUrl()).fastSetValue(true));`

And write `fastSetValue(true)` because you've missed it.